export/policy.ts

```diff
@@ -0,0 +1,72 @@
+import { PhylumApi } from 'phylum';
+
+/**
+ * Helper function for fetching a policy from Phylum.
+ */
+async function fetchPolicyRequest(url: string): Promise<string> {
+    const response = await PhylumApi.fetch("v0/", url, {});
+    
```

Comment:
```suggestion

```
Remove trailing whitespace. There were a few instances of this in `main.ts`, but they are in spots that weren't changed in this review so they can't be commented on in this review.